content/sensu-go/5.0/reference/rbac.md

```diff
@@ -1,85 +1,407 @@
 ---
-title: "RBAC"
-description: "The rbac reference guide."
+title: "Role-based access control"
+linkTitle: "Role-Based Access Control"
+description: "Sensu RBAC reference documentation"
 weight: 1
 version: "5.0"
 product: "Sensu Go"
-platformContent: false 
 menu:
   sensu-go-5.0:
     parent: reference
 ---
 
-- [Organization](#organization)
-- [Environment](#environment)
-- [User](#user)
-- [Group](#group)
-- [Role](#role)
-- [Rule](#rule)
+- [Namespaces](#namespaces)
+- [Resources](#resources)
+- [Users](#users)
+- [Groups](#groups)
+- [Roles and cluster roles](#roles-and-cluster-roles)
+- [Role bindings and cluster role bindings](#role-bindings-and-cluster-role-bindings)
 
-## How does Role Based Access Control work?
-Sensu RBAC allows management and access of users and resources based on a heirarchy of *organizations*, *environments*, *roles*, *rules*, and *users*. Each `environment` belongs to only one `organization`, and each `resource` (user, check, asset, etc) belongs to only one `environment`. A Sensu installation can have multiple organizations, each with their own set of environments and resources belonging to them.
+Sensu role-based access control (RBAC) helps different teams and projects share a Sensu instance.
+RBAC allows management and access of users and resources based on **namespaces**, **roles**, **users**, and **bindings**.
 
-### Authentication
+- **Namespaces** partition resources within Sensu. Sensu checks, handlers, and other [namespaced resources][17] belong to a single namespace.
+- **Roles** create sets of permissions (get, delete, etc.) tied to resource types. **Cluster roles** apply permissions across namespaces and include access to [cluster-wide resources][18] like users and namespaces. 
+- **Users** represent a person or agent that interacts with Sensu. Users can belong to one or more **groups**.
+- **Role bindings** assign a role to a user, and **cluster role bindings** assign a cluster role to a user.
```

Comment:
Two things that might be worth mentioning but not sure how and if it's relevant:

1. Bindings can assign roles to one or multiple users and/or one or multiple groups

2. A role binding can assign both roles and cluster roles to the subjects. In the latter case, what will happen is that the permissions defined in the cluster roles are basically restricted to the role binding's namespace. This is a useful pattern when you want to apply the same permissions across multiple namespaces; you simply create a cluster role and you assign it in the role bindings instead of creating the same role within all these namespaces.

Let me know if you have any questions!